pkg/apis/resource/extension.go

```diff
@@ -242,25 +243,16 @@ func (h Handler) RouteGetAccessEndpoints(req RouteGetAccessEndpointsRequest) (Ro
 					continue
 				}
 
-				var resp *runtime.ResponseCollection
-
-				switch dm.Type {
-				case modelchange.EventTypeCreate:
-					// While create new resource revision,
-					// the previous endpoints from outputs and resources need to be deleted.
-					resp = runtime.TypedResponse(modelchange.EventTypeDelete.String(), eps)
```

Comment:
Do we need to coordinate with UI for this change?

Reply:
Yes, UI is already updated with this change.